charts/signoz/values.yaml

```diff
@@ -225,6 +225,33 @@ clickhouse:
     default/allow_experimental_window_functions: "1"
     default/allow_nondeterministic_mutations: "1"
 
+  # -- Clickhouse init container to copy histogramQuantile UDF
+  # @default -- See `values.yaml` for defaults
+  initContainers:
+    enabled: true
```

Comment:
Why is this whole section here again?

Reply:
The values in clickhouse will only be part of the next clickhouse chart release.

This section is included it here to override the default old values.